src/client/src/services/referenceDataService.ts

```diff
@@ -1,14 +1,14 @@
 import * as _ from 'lodash'
 import { Observable, Scheduler, Subscription } from 'rxjs/Rx'
-import { referenceDataMapper } from './mappers'
-import { ConnectionStatus, UpdateType, ServiceConst } from '../types'
 import { logger, RetryPolicy } from '../system'
-import { ServiceClient } from '../system/service'
 import '../system/observableExtensions/retryPolicyExt'
+import { ServiceClient } from '../system/service'
+import { ConnectionStatus, ServiceConst, UpdateType } from '../types'
+import { referenceDataMapper } from './mappers'
 
 const log = logger.create('ReferenceDataService')
 
-const getReferenceDataStream = (serviceClient, updateCache) => {
+const getReferenceDataStream = (serviceClient, updCache) => {
```

Comment:
I prefer whole words to abbreviations if we have code completion.

Reply:
Alot of these "fixes" are dure to shadowed variable names throughout the code base.